CMakeLists.txt

```diff
@@ -49,6 +49,19 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+if(APPLE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fobjc-arc")
+    if(NOT CMAKE_BUILD_TYPE OR CMAKE_BUILD_TYPE STREQUAL "Debug")
+        if(NOT DEFINED JSRUNTIMEHOST_NATIVE_SANITIZERS)
+            set(JSRUNTIMEHOST_NATIVE_SANITIZERS "address,undefined")
+        endif()
+        if(JSRUNTIMEHOST_NATIVE_SANITIZERS)
+            message(STATUS "macOS sanitizers enabled: ${JSRUNTIMEHOST_NATIVE_SANITIZERS}")
+            add_compile_options("-fsanitize=${JSRUNTIMEHOST_NATIVE_SANITIZERS}" "-fno-omit-frame-pointer")
+            add_link_options("-fsanitize=${JSRUNTIMEHOST_NATIVE_SANITIZERS}")
+        endif()
+    endif()
+endif()
```

Comment:
Are sanitizers supposed to be part of this PR?

Reply:
I was curious about what could be found with sanitizers so I opened this PR : https://github.com/BabylonJS/BabylonNative/pull/1559